gems/aws-sdk-core/spec/aws/log/param_filter_spec.rb

```diff
@@ -55,6 +62,16 @@ class OldServiceType < Struct.new(
           expect(filtered).to eq(sensitive_member: '[FILTERED]')
         end
 
+        it 'filters nested sensitive params' do
```

Comment:
Are map values that are primitive like strings handled? Can we test for that?

Reply:
This was actually an unrelated issue that I discovered while trying to make the code generation fix. 

And yes, maps and documents are handled, but this spec doesn't let us really test for that - we would need specs in the build tools.